README.md

```diff
@@ -10,6 +10,10 @@ Jellyfin Mobile is an Android app built with Cordova.
 ### Getting Started
 
 1. Clone or download this repository.
+   ```git clone https://github.com/jellyfin/jellyfin-android.git
```

Comment:
Could you move this git command to a new line? Otherwise this looks great, thanks.

Reply:
Yeah, this line is invisible, probably because the reasons @dkanada pointed out.